### DISCUSSION TITLE
Drop Area for new search

### DISCUSSION THREAD

**Discussion by katewilga** [yasirkula/UnityAssetUsageDetector#34] [Ideas]
hi, i have a local change that adds a drag and drop area.  

would you be interested in a pull request for this?

![image](https://user-images.githubusercontent.com/6723779/236871269-46e6c377-3530-4854-b907-95a783b7c9cd.png)

**Comment by yasirkula** [comment]
For the time being, I find the "Right click->Search for References" option sufficient but thank you!
